### ISSUE TITLE
[BUG] [Linter] H026 false positive

### ISSUE THREAD

**Issue by OliverGBH** [djlint/djLint#1075]
I've:

- [ X ] I'm on the [latest version](https://pypi.org/project/djlint/) of djLint
- [ X ] I've searched the [issues](https://github.com/djlint/djLint/issues)
- [ X ] I've read the [docs](https://djlint.com)

## System Info

- OS: ubuntu 22.04
- Python Version 3.12
- djLint Version v1.35.2
- template language: Jinja

## Issue

False positive on H026 ruling when linting -- happens when "id" is part of a tag property.

## How To Reproduce
With the following the rule procs:
`<input type="number" name="account_id" placeholder="enter account ID here" />`

despite having no `class` or `id` tag - removing `ID` and `account_ud` fixes the issue linter, but I need those values for those tags.

## Contents of djlint.toml/.djlintrc/pyproject.toml [tool.djlint]

It is to be said I am using pre-commit hooks, whose configuration is as thus:

```json
{
  "indent": 2,
  "ignore": "H030,H031",
  "profile": "jinja"
}

```

with pre-commit hook as thus:
```yaml
- repo: https://github.com/djlint/djLint
  rev: v1.35.2
  hooks:
  - id: djlint-reformat-jinja
  - id: djlint-jinja
```

## Notes

I saw a similar issue here, but it was fixed, I think this might be a related issue: https://github.com/djlint/djLint/issues/711
